### ISSUE TITLE
error status

### ISSUE THREAD

**Issue by zxczxc322** [SagalVanderklok/ChatRoomRecorder#17]
Hello, I would like to know why some accounts have an error status
They are not removed or blocked for my region
![ew](https://github.com/SagalVanderklok/ChatRoomRecorder/assets/37068875/94564ae2-58d5-476f-9c72-d0a5bfad7761)

**Comment by zxczxc322** [comment]
you're right, some accounts are realy removed
I also monitoring another 200+ accounts and some 100% are not deleted and still have an error.
my friend also using this programm and he  told just rerun tool should helps,but for me it doest helps
i think maybe domen name in pasted link matter?

**Comment by zxczxc322** [comment]
i can paste some error nicknames if u want 
and u can check them

**Comment by zxczxc322** [comment]
![cx](https://github.com/SagalVanderklok/ChatRoomRecorder/assets/37068875/d897dd85-4517-4b35-acd5-cce2cd1d94c4)

**Comment by zxczxc322** [comment]
Now i checked those aacounts on second pc
and they have no errors
what could be the problem?

**Comment by SagalVanderklok** [comment]
I'm gonna add extended error messages in the next release. For now try to do the following:
1) remove the problem room from the list, add it again using "bongacams.com" domain name (e.g. https://bongacams.com/kesssik) and start recording with and without VPN
2) close the application, remove "ChatRoomRecorder.exe.WebView2" folder (it locates in a folder with the binary), run the application again and try to start recording

**Comment by zxczxc322** [comment]
almost solved those errors by adding  accounts several times
finally 2 accs left and they didnt work even after 10 times re-adding
I just want to find out what is causing this problem because add same accounts 10 times kind of ruin experience of using this programm

**Comment by SagalVanderklok** [comment]
What are those 2 accounts? Did you use "clear" links like "https://bongacams.com/model" or links copied from the webpage like "https://de.bongacams12.com/model"?

**Comment by zxczxc322** [comment]
just tried clean links and they still have error

**Comment by SagalVanderklok** [comment]
Could you publish these links here?

**Comment by zxczxc322** [comment]
can u also add ability to select timing of monitoring speed
because most of recording starts with huge delay

**Comment by SagalVanderklok** [comment]
Sure, In the coming releases I'll refashion the update algorithm so that rooms will be updated in batches with a pause between such updates (both batch sizes and pause lengths will be configurable separately for different websites). But at first I'll need to fix some bugs recently found

**Comment by Caxapok-X** [comment]
> Hello, I would like to know why some accounts have an error status They are not removed or blocked for my region ![ew](https://private-user-images.githubusercontent.com/37068875/306678074-94564ae2-58d5-476f-9c72-d0a5bfad7761.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxMzMzNTYsIm5iZiI6MTcxMzEzMzA1NiwicGF0aCI6Ii8zNzA2ODg3NS8zMDY2NzgwNzQtOTQ1NjRhZTItNThkNS00NzZmLTljNzItZDBhNWJmYWQ3NzYxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE0VDIyMTczNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRmNGJkYmFkYjJjODI1NTZiMzA3YTY3ZjM0ZTFlN2VhYTc2NjZjOGI4NDc3ODcyM2ZjMWYzNWM0N2Q2YWIzOTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.XKBASw6oqJ-lNYU7BQEIMMviLp40PMiKBS54Qote030)

How did you add them here? I have nothing in this window and I can’t add anyone

**Comment by SagalVanderklok** [comment]
What happens when you add a chat room?

**Comment by Caxapok-X** [comment]
> What happens when you add a chat room?

Nothing happens. When I paste a link to the model in the "web Browser" tab, there is nothing in the "Chat rooms" tab
![pic](https://github.com/SagalVanderklok/ChatRoomRecorder/assets/95716704/05628142-c484-471b-bc80-d3259a1ca739)

**Comment by SagalVanderklok** [comment]
On the "Web browser" tab you should only pass bot protection. After that you should add URLs on the "Chat rooms" tab. There sould be "+" button to the right of the textbox. Do you have it?
By the way, you use an outdated verion (the current version is 1.4.3)

**Comment by Caxapok-X** [comment]
I downloaded all the software from the site, so I was too lazy to download the version from github, now I have updated it to the latest one, on the first tab I am only asked about the return limit. On the second tab I don't have any "+" Perhaps I did not specify the ffmpeg path correctly, did I specify the path to the file, is this path correct?
![pic](https://github.com/SagalVanderklok/ChatRoomRecorder/assets/95716704/e7260106-22ba-4704-986a-877607cf6773)

**Comment by SagalVanderklok** [comment]
You don't see this button, right?
![0](https://github.com/SagalVanderklok/ChatRoomRecorder/assets/122590326/8333ecc0-27bf-4d96-9b4f-098502854727)

**Comment by Caxapok-X** [comment]
> You don't see this button, right? ![0](https://private-user-images.githubusercontent.com/122590326/322446334-8333ecc0-27bf-4d96-9b4f-098502854727.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxNzgzOTIsIm5iZiI6MTcxMzE3ODA5MiwicGF0aCI6Ii8xMjI1OTAzMjYvMzIyNDQ2MzM0LTgzMzNlY2MwLTI3YmYtNGQ5Ni05YjRmLTA5ODUwMjg1NDcyNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQxNVQxMDQ4MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZjM2NmY5MDcxZTgyY2U0NmY2MGRjZmQ3MzdkZDQwMDNiYTJjZjFjZDcyNWI3MzZiZWNkZGUwZjcxNGE3ZDcwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.op9FI_N2zfe4S037zZGnu2oiAis1IhMWoxTUfEvA2o8)

no 🥲🥲🥲

**Comment by SagalVanderklok** [comment]
Did you change DPI settings of your screen?

**Comment by Caxapok-X** [comment]
It automatically opens in full screen, but I still can’t see the “+”
![image](https://github.com/SagalVanderklok/ChatRoomRecorder/assets/95716704/4be6c3ca-153f-4e8b-9b85-8140fd10ac1c)

**Comment by SagalVanderklok** [comment]
Try reset DPI settings (if you changed them) and change desktop theme to default

**Comment by Caxapok-X** [comment]
> Try reset DPI settings (if you changed them) and change desktop theme to default

I had "cleartype" disabled, this solved the problem, thanks for the tip

**Comment by Caxapok-X** [comment]
What is the difference between the records? And do I need to log into my account through the first tab? For example, for private chat entries

**Comment by SagalVanderklok** [comment]
> > Try reset DPI settings (if you changed them) and change desktop theme to default
> 
> I had "cleartype" disabled, this solved the problem, thanks for the tip

Great job! I'll add that "cleartype" issue to my TODO list

> What is the difference between the records? And do I need to log into my account through the first tab? For example, for private chat entries

Sorry, didn't get it. What records do you mean?
If you sign in into your account then I guess you'll be able to record private chats (but I didn't test it though)

**Comment by Caxapok-X** [comment]
I meant this 
![image](https://github.com/SagalVanderklok/ChatRoomRecorder/assets/95716704/0f3211ea-f3f7-4578-9532-bb867c5bf47a)

**Comment by SagalVanderklok** [comment]
In "Monitor" mode the app just monitors room status, while in "Record" mode it captures the stream

**Comment by zxczxc322** [comment]
So in v1.4.4 we can already hope to add the ability to selecting refresh speed?

**Comment by SagalVanderklok** [comment]
> So in v1.4.4 we can already hope to add the ability to selecting refresh speed?

I guess so

**Comment by SagalVanderklok** [comment]
> So in v1.4.4 we can already hope to add the ability to selecting refresh speed?

I've modified the update algorithm in v1.4.4 release. Check it out

**Comment by Caxapok-X** [comment]
I see this topic is still active, so as not to create a new one, I’ll ask here.
Video is recorded in TS format. I encountered a problem where the sound lags behind the event. The delay may be about 50 seconds. I tried to recode the video to another mp4 format but it didn't help. Do I need to install some special codec?

**Comment by SagalVanderklok** [comment]
It's hard to say. I guess there might be a problem on the other side (i.e. with the encoding)

**Comment by Caxapok-X** [comment]
> It's hard to say. I guess there might be a problem on the other side (i.e. with the encoding)
sad

**Comment by zxczxc322** [comment]
> > So in v1.4.4 we can already hope to add the ability to selecting refresh speed?
> 
> I've modified the update algorithm in v1.4.4 release. Check it out
Finally!Will test it now
Whats range of numbers should be used?because there are no restrictions in the line

**Comment by SagalVanderklok** [comment]
Concurrent updates count - from 1 to 99, update interval - from 1 to 86400

**Comment by zxczxc322** [comment]
i mean update interval 
if it set to 1 errors start appearing after refreshing 
at 5 its already 10 times faster than was in 1.4.2
just trying to find optimal numbers

**Comment by zxczxc322** [comment]
and what means 'Concurrent updates'?
max number of recordings(monitoring) account at one time or?

**Comment by SagalVanderklok** [comment]
Chat rooms are updated in steps with the specified update interval. The maximum number of chat rooms that are updated at every step is limited by "cuncurrent updates" settings.

**Comment by zxczxc322** [comment]
thanks for your job man
finally got what i need
perfect tool for me with no lags in recordings and instant monitoring speed

**Comment by SagalVanderklok** [comment]
You're welcome! By the way if you find this app usefull feel free to support the development ;)

**Comment by zxczxc322** [comment]
just minor qol things left 
maybe like support other camsites,ability to copypaste names from chatroom section 
and also save/add multiple accounts from txt.
but for me its almost doesnt matter now.

**Comment by zxczxc322** [comment]
wow just got an error with webview2 click exit and all settings  restored include 300+accounts list
in this case ability to export/import account list from txt really matter.

**Comment by SagalVanderklok** [comment]
> Yeah sure,but it kind of hard make any payment tip from my country also some minor qol advises for next updates like support other camsites,ability to copypaste names from chatroom section import/export account list from txt

I'm planning to implement adding chat rooms in bulk by dragging a previously downloaded file or a folder containing such files into the chat rooms list and also by specifying in the URL textbox a path to a text file containing list of chat rooms

**Comment by zxczxc322** [comment]
any idea why some accounts now shows offline even if they on?(even with vpn)
and also another problem with accounts which blocked from my region
i used vpn in this case and all was fine now they have status record but recording doesn't start
all happened just today and i guess maybe bongacams has changed something

**Comment by zxczxc322** [comment]
if there are no technical problems, it would be nice to add support  recording  from stripchat, in next updates because this is also a large camsite with 10k online models.
I'm currently trying to record using other software and it's not working  very well.

**Comment by SagalVanderklok** [comment]
I'm working on some code and GUI improvements now. Then I'm planning to switch from JSON config file to SQLite database. After that I'm gonna look into adding another websites

**Comment by zxczxc322** [comment]
Recently errors have started appearing during updates.
Most of the time I used 60 Concurrent updates and update interval set at 5
And for a long time everything was relatively stable, but with adding more accounts(over 500), errors began to appear that may not disappear for like 30 mins.
If you slightly change the value even up or down to 1, they start disappear (That's why I think the reason is not the IP address block)
and then after a certain number of updates it happens again.
i already tried diff numbers and still have same problem
any solutions?
the only option I see so far is to reduce the list of added accounts

**Comment by SagalVanderklok** [comment]
There's a bug in the 1.5.1 release which can lead to problems with updating. I've already found the root cause and in the next release the bug will be fixed. For now you can roll back to the 1.5.0 release

**Comment by zxczxc322** [comment]
> There's a bug in the 1.5.1 release which can lead to problems with updating. I've already found the root cause and in the next release the bug will be fixed. For now you can roll back to the 1.5.0 release

but i dont even tried 1,5,1 lol
just testing 1.5 couple days after 1.4
btw what benefits SQLite instead JSON?

**Comment by SagalVanderklok** [comment]
SQLite database is way more reliable and requires far less I/O operations. The problem is that for now there's no way to migrate a chat rooms list from JSON to SQLite

**Comment by zxczxc322** [comment]
Just tried 1.5.2 
most added stripchat accounts have status offline or unknown even if model online 
and still have same status after refreshes

**Comment by SagalVanderklok** [comment]
Hm. I don't have any problems. Check the settings and try to use VPN

**Comment by zxczxc322** [comment]
> Hm. I don't have any problems. Check the settings and try to use VPN
i always use vpn on second pc for stripchat rec
and seems like I figured out what the problem is 
if i trying open account in browser tab it redirect me to domain with region which blocked from model(even with vpn)

**Comment by SagalVanderklok** [comment]
I guess it uses system region/language settings. Try to change that settings in the Control Panel

**Comment by zxczxc322** [comment]
> I guess it uses system region/language settings. Try to change that settings in the Control Panel

u mean change windows language settings?

**Comment by SagalVanderklok** [comment]
> > I guess it uses system region/language settings. Try to change that settings in the Control Panel
> 
> u mean just change windows language settings?

yes

**Comment by SagalVanderklok** [comment]
Or maybe you need to login in the build-in browser into a Stripchat account and change your region there

**Comment by zxczxc322** [comment]
> Or maybe you need to login in the build-in browser into a Stripchat account and change your region there

as expected it doesn't work,will try windows settings

**Comment by zxczxc322** [comment]
after changing language everything working fine but after a couple of weeks an error appears in all accounts if they appear online
can open them normally in browser tab,guessing problem is in the capture method 
i tried different versions of ffmpeg but it didn't help

**Comment by zxczxc322** [comment]
after changing the language everything worked fine for a couple  weeks
but last days those accounts that are currently online have an error status for some reason
and i can open them normally in browser tab
tried different ffmpeg version  but it didnt help

**Comment by zxczxc322** [comment]
weird,after changing basic language everything worked fine for a while
but last week all added accounts have errors status even non blocked with region 
can normally open them in browser tab etc
i think maybe it might be problem with capture method after some update on stripchat?
i tried diff version ffmpeg but in didnt help

**Comment by SagalVanderklok** [comment]
Yeah, they've made some changes in API. I've already fixed that but new version will be released a little bit later as I want to add another fixes that would prevent splitting Stripchat recordings into small pieces

**Comment by zxczxc322** [comment]
cool there are no  freezes in stripchat recordings with streamlink
but a strange thing remained,when I trying to add new account it always have error
It helps me to reset the streamlink directory path then everything works fine

**Comment by SagalVanderklok** [comment]
> cool there are no freezes in stripchat recordings with streamlink but a strange thing remained,when I trying to add new account it always have error It helps me to reset the streamlink directory path then everything works fine

Yeah, there was a little mistake in the code. I've fixed this in the 1.5.4 release

**Comment by zxczxc322** [comment]
does stripchat changed api again?
i have errors status on all online accounts

**Comment by SagalVanderklok** [comment]
They've just made some changes to API again. I've already fixed it but the next version will be a little bit later

**Comment by zxczxc322** [comment]
can u just make hotfix for current patch?

**Comment by Caxapok-X** [comment]
![124](https://github.com/user-attachments/assets/04600714-bf34-4ccd-9b1b-88d1c1675995)
Tell me why I get this message although the link is correct
The URL is incorrect!

**Comment by SagalVanderklok** [comment]
Because you need to specify a URL to a chat room (e.g. https://de.bongacams.com/gabriellalee)

**Comment by Caxapok-X** [comment]
> Because you need to specify a URL to a chat room (e.g. https://de.bongacams.com/gabriellalee)
It's all right there was one “/” missing at the very end of the link
https://de.bongacams.com/gabriellalee/  
That's how I did it.

**Comment by SagalVanderklok** [comment]
Nope, I can add a chat room using both that URLs (with and without trailing /)

**Comment by SagalVanderklok** [comment]
@Caxapok-X  Could you please try to add a URL in v1.6.8? I can't reproduce the error, but other users can also encounter this problem

**Comment by Caxapok-X** [comment]
> [@Caxapok-X](https://github.com/Caxapok-X) Could you please try to add a URL in v1.6.8? I can't reproduce the error, but other users can also encounter this problem

At the moment, I am using version 1.6.2.

**Comment by SagalVanderklok** [comment]
Thanks! That would be great!

**Comment by Caxapok-X** [comment]
![Image](https://github.com/user-attachments/assets/a1a86ce4-d71d-4dc6-ae49-9b0d1fee2998)
I apologize for the wait, but I was only able to check just now. I also can't add the link even when using different options.

**Comment by SagalVanderklok** [comment]
Thanks a lot! The root cause of the error is that current version supports only "com" domains for BongaCams. I'll add support of "cam" domains in the next release

**Comment by Caxapok-X** [comment]
> Thanks a lot! The root cause of the error is that current version supports only "com" domains for BongaCams. I'll add support of "cam" domains in the next release (for now you can just replace "bongacams.cam" with "bongacams.com" in order to add chat rooms)

Damn, I’m so inattentive that I didn’t notice this issue. 🤣

**Comment by SagalVanderklok** [comment]
Support for "bongacams.cam" domain has been added in v.1.6.9
